CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 #Changelog for the fact-tools
 
+# Version 0.17.4 -- 17.01.2017
+
+* add fact.utils.PhotonStream2ArrivalTime
```

Comment:
Idont like the location fact.utils. Can we move this to its own package fact.photonStream?